tests/db_integrity/test_data_updates.py

```diff
@@ -6,108 +6,80 @@
 from openghg.store.base import Datasource
 from openghg.retrieve import get_flux
 from openghg.retrieve import search
-from openghg.standardise import standardise_footprint
+from openghg.standardise import standardise_footprint, standardise_flux, standardise_surface
 from openghg.objectstore import get_bucket
-
-
 from helpers import clear_test_stores
 
 
-def flux_data_read():
-    """
-    Flux data set up.
+@pytest.fixture(autouse=True)
```

Comment:
I think we should be careful with overusing `autouse` as it can cause confusion - my feel is that for these tests in particular we'd be better off being more explicit with clearing the store at the start of each test to make it clear the steps that have been taken.

Reply:
@brendan-m-murphy - just a lingering comment from previous review on the use of autouse for clear_stores in test_data_updates.py to address.